app/src/main/java/com/fstyle/structure_android/data/source/local/config/sqlite/UserDbHelper.java

```diff
@@ -35,10 +37,26 @@
             + TEXT_TYPE
             + " )";
 
+    private static UserDbHelper INSTANCE;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName